modules/oxide/src/less/theme/content/selection/selection.less

```diff
@@ -83,6 +83,19 @@
     }
   }
 
+  // Apply cell selection as a semi-transparent overlay instead of manipulating the cell background itself
+  td[data-mce-selected]::after,
+  th[data-mce-selected]::after {
+    background-color: @table-selection-color;
```

Comment:
Do we normally write CSS properties in alphabetical order? If that's a convention then that's fine, but if not could we group them by similar properties — put bottom / top / left / right all together, background-colour / opacity, etc..

Reply:
If you look at the stylelint rules you'll see it's required to write them in alphabetical order.